From 4bfaf6f4b41f18824e2a251026ada4a5b69221d1 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 2 Feb 2024 14:08:14 +0000
Subject: [PATCH] overlays: Correct some compatible strings

More thorough overlay testing has identified some Pi 4-specific
overlays that has "brcm,bcm2835" compatible strings. Correct them.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/midi-uart2-overlay.dts  | 2 +-
 arch/arm/boot/dts/overlays/midi-uart3-overlay.dts  | 2 +-
 arch/arm/boot/dts/overlays/midi-uart4-overlay.dts  | 2 +-
 arch/arm/boot/dts/overlays/midi-uart5-overlay.dts  | 2 +-
 arch/arm/boot/dts/overlays/ramoops-pi4-overlay.dts | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/midi-uart2-overlay.dts b/arch/arm/boot/dts/overlays/midi-uart2-overlay.dts
index 66f3092e9a74..5c6985f41ea2 100644
--- a/arch/arm/boot/dts/overlays/midi-uart2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/midi-uart2-overlay.dts
@@ -12,7 +12,7 @@
  */
 
 /{
-        compatible = "brcm,bcm2835";
+        compatible = "brcm,bcm2711";
 
         fragment@0 {
                 target-path = "/";
diff --git a/arch/arm/boot/dts/overlays/midi-uart3-overlay.dts b/arch/arm/boot/dts/overlays/midi-uart3-overlay.dts
index 55c6cb94f963..052027db0564 100644
--- a/arch/arm/boot/dts/overlays/midi-uart3-overlay.dts
+++ b/arch/arm/boot/dts/overlays/midi-uart3-overlay.dts
@@ -12,7 +12,7 @@
  */
 
 /{
-        compatible = "brcm,bcm2835";
+        compatible = "brcm,bcm2711";
 
         fragment@0 {
                 target-path = "/";
diff --git a/arch/arm/boot/dts/overlays/midi-uart4-overlay.dts b/arch/arm/boot/dts/overlays/midi-uart4-overlay.dts
index 5819df1a6b2e..5f09a7ccd675 100644
--- a/arch/arm/boot/dts/overlays/midi-uart4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/midi-uart4-overlay.dts
@@ -12,7 +12,7 @@
  */
 
 /{
-        compatible = "brcm,bcm2835";
+        compatible = "brcm,bcm2711";
 
         fragment@0 {
                 target-path = "/";
diff --git a/arch/arm/boot/dts/overlays/midi-uart5-overlay.dts b/arch/arm/boot/dts/overlays/midi-uart5-overlay.dts
index a1d37f7103ff..74551ec2a672 100644
--- a/arch/arm/boot/dts/overlays/midi-uart5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/midi-uart5-overlay.dts
@@ -12,7 +12,7 @@
  */
 
 /{
-        compatible = "brcm,bcm2835";
+        compatible = "brcm,bcm2711";
 
         fragment@0 {
                 target-path = "/";
diff --git a/arch/arm/boot/dts/overlays/ramoops-pi4-overlay.dts b/arch/arm/boot/dts/overlays/ramoops-pi4-overlay.dts
index 4f3d30ef069a..1737e37f5724 100644
--- a/arch/arm/boot/dts/overlays/ramoops-pi4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ramoops-pi4-overlay.dts
@@ -2,7 +2,7 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2835";
+	compatible = "brcm,bcm2711";
 
 	fragment@0 {
 		target = <&rmem>;
-- 
2.34.1

